typescript/.changeset/cyan-cooks-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/agentkit": patch
+---
+
+Fixed under-reporting CDP API metrics bug by properly adding source and source version when configuring the CDP SDK
```

Comment:
@John-peterson-coinbase Is this change relevant to include in the changelog? I think we can probably omit it since it's really just for us